src/catalog/src/system_schema/information_schema.rs

```diff
@@ -299,3 +302,12 @@ where
         InformationTable::to_stream(self, request)
     }
 }
+
+/// Provide query context to call the [`CatalogManager`]'s method.
+static MYSQL_QUERY_CTX: LazyLock<QueryContext> = LazyLock::new(|| {
+    QueryContext::with_channel(DEFAULT_CATALOG_NAME, DEFAULT_SCHEMA_NAME, Channel::Mysql)
+});
+
+fn query_ctx() -> Option<&'static QueryContext> {
+    Some(&MYSQL_QUERY_CTX)
+}
```

Comment:
I think the only tricky point is here. I still suggest to use `None` query context for `information_schema`'s builder where there is no real query context available. 

Different from `pg_catalog`, which is dedicated to postgres, `information_schema` is a global concept that available to all protocols like mysql, postgresql and others we offer. So I think it's safe to use `None` and it's default behaviour here. 

Using `Mysql` channel may introduce misunderstanding in future. It's hard to distinguish whether it's real mysql or placeholder.

Reply:
fair enough. done.